app/uk/gov/hmrc/pillar2/repositories/RegistrationCacheRepository.scala

```diff
@@ -81,58 +65,53 @@ class RegistrationCacheRepository @Inject() (
       collectionName = "user-answers-records",
       mongoComponent = mongoComponent,
       domainFormat = RegistrationDataEntryFormats.format,
-      extraCodecs = Seq(
-        Codecs.playFormatCodec(JsonDataEntry.format),
-        Codecs.playFormatCodec(DataEntry.format)
-      ),
       indexes = Seq(
         IndexModel(
-          Indexes.ascending(expireAtKey),
+          Indexes.ascending(lastUpdatedKey),
           IndexOptions()
-            .name("dataExpiry")
-            .expireAfter(2419200, TimeUnit.SECONDS)
-            .background(true)
+            .name("lastUpdatedIndex")
+            .expireAfter(config.defaultDataExpireInDays, TimeUnit.DAYS)
```

Comment:
Was there any particular reason to change in days. having in seconds it can be easily tested?

Reply:
Have we tested lastUpdatedIndex for records to be deleted? Manual test may be required.

Reply:
This was manually tested. I don't think it would make it any difficult to test if its days but I can change it back to seconds if you think its a better option @AKKUM

Reply:
Only thing I am thinking if someone update 10PM, will that be counted 1 day? if that is in seconds, it may be accurate 28 days? Tried to look other services, most of them looks seconds. Happy to sign it off.

Reply:
you are right. Changed back to seconds